packages/medusa/src/loaders/medusa-app.ts

```diff
@@ -0,0 +1,58 @@
+import { MedusaApp, ModulesDefinition } from "@medusajs/modules-sdk"
```

Comment:
praise(this-file): nice

Reply:
Praise too, will have lot of conflicts as it will change quite a lot with out pr which is something we will probably extract to a separate pr 😅 as there is some issues